.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,25 @@
+## Description
+
+Please include a summary of the change, motivation and context
+
+- **on a feature**: describe the feature and how this change fits in it, e.g. this PR makes kafka message.max.bytes configurable to better support batching
+- **on a refactor**: describe why this is better than previous situation e.g. this PR changes logic for retry on healthchecks to avoid false positives
+- **on a bugfix**: link relevant information about the bug (github issue or slack thread) and how this change solves it e.g. this change fixes #99999 by adding a lock on read/write to avoid data races.
+
+### Testing
+
+Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration
+
+- [ ] Docker-compose
+- [ ] Helm install
+
+### Checklist:
+- [ ] My changes generate no new warnings
+- [ ] I have added tests that prove my fix is effective or that my feature works
+- [ ] Any dependent changes have been merged and published in downstream modules
+
+### Documentation
+Make sure that you have made corresponding changes to the documentation in README and [hypertrace docs repo](https://github.com/hypertrace/hypertrace-docs-website) if required.  
```

Comment:
nit: make this more generic:

Make sure that you have documented corresponding changes in this repository or in [hypertrace docs repo](https://github.com/hypertrace/hypertrace-docs-website) if required.